src/services/AddressService.jsx

```diff
@@ -0,0 +1,10 @@
+const getFullAddress = address => {
```

Comment:
after looking at your tests, it looks like address you are passing in array, but you should only need to pass in the object

example

currently 
```js
[{
  address1: "123 Abc Ave"
}]
```

proposed

```js
{
  address1: "123 Abc Ave"
}
```

Reply:
this will impact your tests, so they will need to be updated

Reply:
change code to pass in the object instead of array